app/components/job-authorize.js

```diff
@@ -1,7 +1,33 @@
 import Ember from 'ember';
 
 const JobAuthorize = Ember.Component.extend({
-  token: '',
+  tagName: 'div',
+  classNames: ['job-authorize'],
+  job: null,
+  _token: '',
+  token: Ember.computed('job.needsAuthorization', 'job.runToken', '_token', {
```

Comment:
I think the computed property is `hasAuthorization` not `needsAuthorization`.

Reply:
Thanks, was just about to sit down and debug this :)